src/Plugin/video_embed_field/Provider/GoogleDrive.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\stanford_media\Plugin\video_embed_field\Provider;
+
+use Drupal\video_embed_field\ProviderPluginBase;
+
+/**
+ * A YouTube provider plugin.
+ *
+ * @VideoEmbedProvider(
+ *   id = "google_drive",
+ *   title = @Translation("Google Drive")
+ * )
+ */
+class GoogleDrive extends ProviderPluginBase {
```

Comment:
As this is specific to GoogleDrive video do we want to rename this? Perhaps we would have GoogleDrive document integration in the future.

Reply:
its only for video embed providers. similar to the `youtube` provider. any document plugin such as the ones provided by https://www.drupal.org/project/media_entity_googledocs wont have any conflicts because of the annotation `@VideoEmbedProvider`

Reply:
Thanks for the explanation.